guessinggame.py

```diff
@@ -0,0 +1,25 @@
+import random
+
+number = random.randint(0, 10)
+guess = int(input("Enter an integer from 0 to 10: "))
+numofguesses = 0
+
+while number != guess:
+    print ("Sorry, try again!")
+    guess = int(input("Enter an integer from 0 to 10: "))
+    numofguesses = numofguesses + 1
```

Comment:
What if the user guesses the correct number on the 5th try? Why does he/she still get the "Do you want a hint message?"

Reply:
I'll add the condition "if numofguesses == 4 and number != guess"